examples/bearriver/src/FRP/BearRiver.hs

```diff
@@ -188,6 +188,43 @@ occasionally tAvg b
   timeDelta :: Monad m => SF m a DTime
   timeDelta = arrM_ ask
 
+-- | Performs uniform super sampling on the provided signal-function with n samples
+-- 
+-- This is done by dividing the current time-delta into N equally spaced samples 
+-- and evaluating the signal function with these subsamples and always the same input. 
+-- The first argument is the number of samples - if LT 0 exactly 1 sample is calculated
```

Comment:
Can you rather use haddock function argument comments? See e.g. https://www.haskell.org/haddock/doc/html/ch03s02.html#idm140354810892448 (3.2.3. Function arguments)